spec/views/idv/in_person/ready_to_verify/show.html.erb_spec.rb

```diff
@@ -84,86 +67,125 @@
-      )
+  context 'outage alert' do
+    context 'when the outage message flag is enabled' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_outage_message_enabled).
+          and_return(true)
+      end
+
+      context 'when the outage dates are included' do
+        let(:formatted_date) { 'Tuesday, October 31' }
+        let(:in_person_outage_emailed_by_date) { 'November 1, 2023' }
+        let(:in_person_outage_expected_update_date) { 'October 31, 2023' }
+
+        before do
+          allow(IdentityConfig.store).to receive(:in_person_outage_message_enabled).
+            and_return(true)
+          allow(IdentityConfig.store).to receive(:in_person_outage_emailed_by_date).
+            and_return(in_person_outage_emailed_by_date)
+          allow(IdentityConfig.store).to receive(:in_person_outage_expected_update_date).
+            and_return(in_person_outage_expected_update_date)
+        end
+
+        it 'renders the outage alert' do
+          render
+
+          expect(rendered).to have_content(
+            t(
+              'idv.failure.exceptions.in_person_outage_error_message.ready_to_verify.title',
+              date: formatted_date,
+            ),
+          )
+        end
+      end
+
+      context 'when the outage dates are not included' do
+        before do
+          allow(IdentityConfig.store).to receive(:in_person_outage_message_enabled).
+            and_return(true)
+          allow(IdentityConfig.store).to receive(:in_person_outage_emailed_by_date).
+            and_return('')
+          allow(IdentityConfig.store).to receive(:in_person_outage_expected_update_date).
+            and_return('')
+        end
+
+        it 'does not render a warning' do
+          render
+
+          expect(rendered).to_not have_content(
+            t('idv.failure.exceptions.in_person_outage_error_message.ready_to_verify.title'),
+          )
+        end
+      end
     end
 
-    it 'does not render the outage alert when flag is disabled' do
-      allow(IdentityConfig.store).to receive(:in_person_outage_message_enabled).
-        and_return(false)
+    context 'when the outage message flag is disabled' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_outage_message_enabled).
+          and_return(false)
+      end
 
-      render
+      it 'does not render the outage alert' do
+        render
 
-      expect(rendered).not_to have_content(
-        t('idv.failure.exceptions.in_person_outage_error_message.ready_to_verify.title'),
-      )
+        expect(rendered).not_to have_content(
+          t('idv.failure.exceptions.in_person_outage_error_message.ready_to_verify.title'),
+        )
+      end
     end
   end
 
-  context 'For IPP (Not Enhanced IPP)' do
-    let(:is_enhanced_ipp) { false }
-    before do
-      @is_enhanced_ipp = false
-    end
+  context 'what to expect section' do
+    context 'when Enhanced IPP is not enabled' do
+      let(:is_enhanced_ipp) { false }
```

Comment:
Non-blocking: I wonder if [Andrew's comment on the user mailer spec](https://github.com/18F/identity-idp/pull/11520#discussion_r1846901355) also applies here?